SkeletonProject/page_object/page.py

```diff
@@ -0,0 +1,76 @@
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class BasePage(object):
+
+    SELENIUM_TIMEOUT_SECONDS = 5
+
+    def __init__(self, driver):
+        self.driver = driver
+
+
+    def get_max_size_window(self):
```

Comment:
why do you add this method in BasePage ? is it reused somewhere ? Please remove this method.